service/src/main/resources/templates/ehr_request_statement_template.mustache

```diff
@@ -1,26 +1,26 @@
 <component typeCode="COMP" {{#isNested}}contextConductionInd="true"{{/isNested}}>
     <RequestStatement classCode="OBS" moodCode="RQO">
         <id root="{{requestStatementId}}" />
+        {{#code}}
+        {{{code}}}
+        {{/code}}
+        <text>{{text}}</text>
         <effectiveTime>
             <center nullFlavor="NI"/>
         </effectiveTime>
         {{{availabilityTime}}}
-        {{#defaultReasonCode}}
-        {{{defaultReasonCode}}}
-        {{/defaultReasonCode}}
-        <text>{{text}}</text>
-        {{#code}}
-        {{{code}}}
-        {{/code}}
-        {{#participant}}
-        {{{.}}}
-        {{/participant}}
         {{#responsibleParty}}
         <responsibleParty typeCode="RESP">
             <agentRef classCode="AGNT">
                 <id root="{{{.}}}"/>
             </agentRef>
         </responsibleParty>
         {{/responsibleParty}}
+        {{#participant}}
+        {{{.}}}
+        {{/participant}}
+        {{#defaultReasonCode}}
```

Comment:
Believe defaultReasonCode is used when code is not populated.
Thus think this should be moved to be above text element tag.